HW0/P5/P5.txt

```diff
@@ -1 +1,79 @@
 # Your answers here.
```

Comment:
Your graph for 5 doesn't seem to imply any trends in performance (both look linear) and it's hard to make out that in parallel it would be logarithmic.